lib/sedna/datasources/__init__.py

```diff
@@ -130,3 +130,23 @@ def parse(self, *args, **kwargs):
             return
         self.x = pd.concat(x_data)
         self.y = pd.concat(y_data)
+
+# import os
+# os.environ['config_file'] = '/home/work/client.yml'
```

Comment:
it is recommended to clean up unwanted codes.

Reply:
updated